addons/payment/tests/test_payment_method.py

```diff
@@ -142,3 +143,28 @@ def test_payment_method_not_compatible_with_express_checkout(self):
             self.provider.ids, self.partner.id, is_express_checkout=True
         )
         self.assertNotIn(self.payment_method, compatible_payment_methods)
+
+    def test_payment_method_availability_report(self):
+        """ Test that report values are correctly set when the payment method is available and
+        when it is unavailable."""
```

Comment:
```suggestion
        """ Test that report values are correctly set when the payment method is available and when 
        it is unavailable."""
```
You don't have to break sooner than 100 chars =]